grpclb/src/main/java/io/grpc/grpclb/GrpclbState.java

```diff
@@ -259,11 +259,16 @@ void handleAddresses(
         serviceName,
         newLbAddressGroups,
         newBackendServers);
+    fallbackBackendList = newBackendServers;
     if (newLbAddressGroups.isEmpty()) {
       // No balancer address: close existing balancer connection and enter fallback mode
```

Comment:
Let's update this comment as well, because it disagrees with the code.

Reply:
Updated the comment to more precisely describe the behavior and considerations are are facing.

Reply:
Thanks, LGTM